pkg/config/engine/containerd/containerd.go

```diff
@@ -162,8 +162,11 @@ func (c *Config) GetRuntimeConfig(name string) (engine.RuntimeConfig, error) {
 }
 
 // CommandLineSource returns the CLI-based containerd config loader
-func CommandLineSource(hostRoot string) toml.Loader {
-	return toml.FromCommandLine(chrootIfRequired(hostRoot, "containerd", "config", "dump")...)
+func CommandLineSource(hostRoot string, executablePath string) toml.Loader {
+	if executablePath == "" {
+		executablePath = "containerd"
```

Comment:
Probably tangential : So far, we've made the assumption that the `containerd` binary will be resolved via `$PATH`. Would it make sense to look up a set of known absolute paths of the containerd binary until we git a hit? 

I see that you already have a method to do exactly just this: https://github.com/NVIDIA/nvidia-container-toolkit/blob/1d2e1bd403102262beea909ffc99d26f294f084f/internal/lookup/path.go#L36

Reply:
That could make sense. Do you have a list of known paths that you would like to check?

For reference the "executable" locator that uses `GetPaths` also checks the `PATH` variable to determine which locations to check (in addition to the default values). I didn't want to change that behavior as part of this change though.

Reply:
Agreed